articles/node-js-vs-django/index.md

```diff
@@ -0,0 +1,83 @@
+5. Portable – There is no need to write different codes for different machines as it is highly portable and flexible.
+6. It is available with a vast collection of libraries – Libraries are inbuilt, and therefore, there is no need to write unique code for everything.
+
+### Comparison between Django and Node.js
+
+| Distinctive point | Django | Node.js |
+| --- | --- | --- |
+| Programming language and primary use | It is a [Python based framework](https://www.upgrad.com/blog/python-frameworks/) used by developers to build computer apps within a stipulated time frame.| A framework that runs on JavaScript. Developers use it to build robust application programming interfaces (client-sided). |
+| Architecture | Follows the [MTV model template](https://www.javatpoint.com/django-mvt) for purposes of handling data, validating, and interaction. | A runtime environment, therefore, works on an [event-driven model](https://www.redhat.com/en/topics/integration/what-is-event-driven-architecture). Runs on any operating system, maintaining a small number of requests. |
+| Security | It is more secure, comes with a built-in system, preventing a security breach. | It requires manual operations systems to manage security risks. |
+| Performance | It has a built-in house template system; hence the execution of tasks is quicker, thereby offering better performance. | It allows web professionals freedom in implementations. Though, it has a limitation of increasing the time required to build an application. |
+| Complexity | It is more complicated as a developer has to follow a particular specified path for solving problems. | It is less complicated as a developer has a free hand to solve problems. |
+| Efficiency | More efficient and faster, hence cost-effective. | It consumes more operation time; hence it is a less cost-effective option. |
+| Reputation | It has a good and solid reputation. | Its popularity is growing with time. Soon it might become a preferred framework. |
+| Users or the community | It has a reasonably active, though small community. | It has quite an active community with experienced users that assist with updates and customization. |
+| Flexibility | It has strict features and does not offer flexibility. | A variety of tools and features exists. One can build an app from scratch, and therefore it is more flexible. |
+
+### Advantages and disadvantages of using Django
+
+#### Advantages
+
+- It is comparatively easy to learn.
+- Its built-in security system ensures security risks are very low.
+- It ensures app and web development is done at a fast rate and hence cost-effective.
+- The level of performance in the in-built templates is high.
+
+#### Disadvantages
+
+- It is not flexible, allows only [single-tiered](https://docs.bitnami.com/google-templates/singletier-vs-multitier/) applications.
+- It has limitations with small apps. It does not work well in small apps.
+- It requires much understanding of the framework.
```

Comment:
much understanding -> detailed understanding. 
Also, isn't that the case with Node.js as well?